pkg/utils/utils.go

```diff
@@ -0,0 +1,56 @@
+package utils
```

Comment:
we can find a better place for the following function - just a proposition.

Reply:
We should avoid a `utils` package. Instead we should define explicit packages.

For `GetClusterInfoKubeconfig` i would create a new package called `clusterinfo`
For `ReadinessChecks` i would move it into the `health` package

Reply:
why not, sounds good to me.

Reply:
The reason against using a `utils` is mostly because it normally becomes a mess where people put code it, they don't know where to put else. It becomes unorganized and messy.

A good read on that is: https://blog.golang.org/package-names

Reply:
I think that we don't actually need another package at all. 
I'd put the `ReadinessChecks` simply into the main.go file. That's the only place this is used. There's no need to put it into another package. 

With the `GetClusterInfoKubeconfig` I'm not entirely sure, because it's use in `ReadinessChecks` and somewhere in the controller AFAIK.

Thoughts?

Reply:
@metalmatze i agree